pygmt/src/grdview.py

```diff
@@ -126,7 +126,9 @@ def grdview(self, grid, **kwargs):
         file_context = lib.virtualfile_from_data(check_kind="raster", data=grid)
 
         with contextlib.ExitStack() as stack:
-            if "G" in kwargs:  # deal with kwargs["G"] if drapegrid is xr.DataArray
+            if (
+                kwargs.get("G") is not None
+            ):  # deal with kwargs["G"] if drapegrid is xr.DataArray
```

Comment:
Writing the comment in this way makes the code look better.
```suggestion
            if kwargs.get("G") is not None:  
                # deal with kwargs["G"] if drapegrid is xr.DataArray
```